### PR TITLE
singular_values_ attribute for MultipleASE class

### LINKED ISSUE #548: singular_values_ attribute needed for MultipleASE class
## Expected Behavior
MultipleASE should have an attribute that stores singular values for ease of generating scree plots, consistent with the other embedding classes.  Specifically, it should output the singular values of the concatenated adjacency spectral embeddings, consistent with Figure 10 of the [paper](https://arxiv.org/pdf/1906.10026.pdf) where it is introduced.

## Actual Behavior
MultipleASE has no attribute for singular values.

## Template Code
N/A

## Full Traceback
N/A

## Your Environment
* Python version: 3.7.6
* graspologic version: 0.1.0

## Proposed Labels
 * Question

### PR DESCRIPTION
**Pull request by pauladkisson**
#### Reference Issues/PRs
Fixes #548 

#### What does this implement/fix? Explain your changes.
This update adds a singular_values_ attribute to the MultipleASE class, which is equal to the singular values of the concatenated adjacency spectral embeddings.  When input graphs are directed, singular_values_ is a tuple (left singular values, right singular values).

#### Any other comments?
I think that the changes to setup.py are from black.

### CODE REVIEW

File: graspologic/embed/mase.py [hunk 1]
```diff
@@ -87,6 +87,10 @@ class MultipleASE(BaseEmbedMulti):
     scores_ : array, shape (n_samples, n_components, n_components)
         Estimated :math:`\hat{R}` matrices for each input graph.
 
+    singular_values_ : array, shape (n_components) OR length 2 tuple of arrays
+        If input graph is undirected, equal to the singular values of the concatenated adjacency spectral embeddings.
+        If input graph is directed, :attr:`singular_values_` is a tuple of length 2, where :attr:`singular_values_[0]` corresponds to the concatenated left
```
Comment (bdpedigo) [inline graspologic/embed/mase.py:92]: make sure this is <88 chars? if it is please ignore/resolve this comment.

File: graspologic/embed/mase.py [hunk 2]
```diff
@@ -87,6 +87,10 @@ class MultipleASE(BaseEmbedMulti):
...
+        adjacency spectral embeddings, and :attr:`singular_values_[1]` corresponds to the concatenated right adjacency spectral embeddings.
```
Comment (bdpedigo) [inline graspologic/embed/mase.py:93]: "concatenated right adjacency spectral embeddings"
I would say singular values of the concatenated ...

**Review by bdpedigo — APPROVED** [review]
i had two very minor comments

@j1c any thoughts here? Only think I can think of is whether we want the tuple in the directed  case (as opposed to 2 attributes) but I think it makes sense here

File: graspologic/embed/mase.py (hunk 1 above)
Comment (pauladkisson) [reply graspologic/embed/mase.py:92]: Will fix

File: graspologic/embed/mase.py (hunk 2 above)
Comment (pauladkisson) [reply graspologic/embed/mase.py:93]: Will do

**Comment by pauladkisson** [comment]
The only thing I was a little worried about is that the description for singular_values_ is somewhat longer than all the other attributes.

**Comment by bdpedigo** [comment]
@pauladkisson are you aware of these test fails?

**Comment by pauladkisson** [comment]
>are you aware of these test fails?

@bdpedigo, I'm not sure what's going on with those because when it says "the term pytest is not recognized" but that doesn't have anything to do with my changes.

**Pull request merged**